day_6/exercises/person.js

```diff
@@ -8,4 +8,25 @@ person methods below the class so that they print their result to the
 terminal.
 */
 
-// YOUR CODE HERE
+class Person {
+  constructor(weight, name, nationality) {
+    this.weight = weight;
+    this.name = name;
+    this.nationality = nationality;
+  }
+
+  grow(gain) {
+    var gain = this.weight++;
```

Comment:
Note that your parameter and var are the same here. How is a parameter typically used?